testing/test_conftest.py

```diff
@@ -494,29 +497,29 @@ def test_no_conftest(fxtr):
         ],
     )
     def test_parsefactories_relative_node_ids(
-        self, testdir, chdir, testarg, expect_ntests_passed
-    ):
+        self, pytester: Pytester, chdir, testarg, expect_ntests_passed
+    ) -> None:
         """#616"""
-        dirs = self._setup_tree(testdir)
-        print("pytest run in cwd: %s" % (dirs[chdir].relto(testdir.tmpdir)))
+        dirs = self._setup_tree(pytester)
```

Comment:
> testing/test_conftest.py:501: error: "_setup_tree" of "TestConftestVisibility" does not return a value [func-returns-value]

I believe this error is because of last error you mentioned.
> testing/test_conftest.py:468: error: No return value expected [return-value]

Fixing that error will resolve this error.